packages/documentation/src/stories/components/popover/popover.stories.ts

```diff
@@ -0,0 +1,94 @@
+import { Args, Meta, StoryObj } from '@storybook/web-components';
+import { BADGE } from '../../../../.storybook/constants';
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+const meta: Meta = {
+  title: 'Components/Popover',
+  component: 'post-popover',
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+    docs: {
+      argTypes: {
+        sort: 'requiredFirst',
+      },
+    },
+  },
+  render,
+  args: {
+    id: 'popover-one',
+    innerHtml:
+      'A longer message that needs more time to read. <a href="#">Links</a> are also possible.',
+    backgroundColor: 'primary',
+    closeButtonCaption: 'Close',
+    placement: 'top',
+    arrow: true,
+  },
+  argTypes: {
+    id: {
+      name: 'Id',
+      description:
+        'The id is used to connect a trigger element with the popover. <div className="mt-mini alert alert-info alert-sm">`<button data-popover-target="...">` is the only valid trigger element for `post-popover`.</div>',
+      table: {
+        category: 'General',
+      },
+    },
+    innerHtml: {
+      name: 'Content',
+      table: {
+        category: 'General',
+      },
+    },
+    backgroundColor: {
+      name: 'Background color',
+      description: 'Define a background color, either `bg-primary` or `bg-yellow`.',
+      control: {
+        type: 'radio',
+        labels: {
+          yellow: 'Yellow',
+          primary: 'Primary',
+        },
+      },
+      options: ['primary', 'yellow'],
+      table: {
+        category: 'General',
+        type: {
+          summary: 'HTML class attribute',
+        },
+      },
+    },
+    closeButtonCaption: {
+      name: 'Close button caption',
+      table: {
+        type: {
+          required: true,
+        },
+      },
+    },
+    arrow: {
+      name: 'Arrow',
+    },
+    placement: {
+      name: 'Placement',
+    },
+  },
+};
+
+function render(args: Args) {
+  return html`
+    <button class="btn btn-secondary btn-large" data-popover-target="${args.id}">
+      Click here to see a popover
+    </button>
+    <post-popover
+      class="hydrated bg-${args.backgroundColor}"
+      id="${args.id}"
+      placement="${args.placement}"
+      ?arrow="${args.arrow}"
+    >
+      ${unsafeHTML(args.innerHtml)}
```

Comment:
If you don't wrap the content with an element, the selection of the text is limited in Firefox (and chrome, but differently). Triple click doesn't select all content in Firefox. In chrome, it will only select a sentence.

Reply:
Done